<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QKflow</class>
 <widget class="QMainWindow" name="QKflow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QkFlow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="PnView" name="pnView" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>25</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomFit"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomFit"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSlack"/>
   <addaction name="actionPQBar"/>
   <addaction name="actionPVBar"/>
   <addaction name="separator"/>
   <addaction name="actionCable"/>
   <addaction name="actionTransformer"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New Power Network project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Network</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Network</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/run.png</normaloff>:/img/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run simulation</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/pause.png</normaloff>:/img/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause simulation</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/stop.png</normaloff>:/img/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop simulation</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/settings.png</normaloff>:/img/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Configure simulation parameters</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/zoom+.png</normaloff>:/img/zoom+.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/zoom-.png</normaloff>:/img/zoom-.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoomFit">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/search.png</normaloff>:/img/search.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Fit</string>
   </property>
   <property name="toolTip">
    <string>Fit screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSlack">
   <property name="icon">
    <iconset resource="PnGraphics/pnrc.qrc">
     <normaloff>:/img/ico-slack.png</normaloff>:/img/ico-slack.png</iconset>
   </property>
   <property name="text">
    <string>Slack</string>
   </property>
   <property name="toolTip">
    <string>Add Slack</string>
   </property>
  </action>
  <action name="actionPQBar">
   <property name="icon">
    <iconset resource="PnGraphics/pnrc.qrc">
     <normaloff>:/img/ico-pq.png</normaloff>:/img/ico-pq.png</iconset>
   </property>
   <property name="text">
    <string>PQBar</string>
   </property>
   <property name="toolTip">
    <string>Add PQBar</string>
   </property>
  </action>
  <action name="actionTransformer">
   <property name="icon">
    <iconset resource="PnGraphics/pnrc.qrc">
     <normaloff>:/img/ico-trans.png</normaloff>:/img/ico-trans.png</iconset>
   </property>
   <property name="text">
    <string>Trans...</string>
   </property>
   <property name="toolTip">
    <string>Add Transformer</string>
   </property>
  </action>
  <action name="actionCable">
   <property name="icon">
    <iconset resource="PnGraphics/pnrc.qrc">
     <normaloff>:/img/ico-line.png</normaloff>:/img/ico-line.png</iconset>
   </property>
   <property name="text">
    <string>Cable</string>
   </property>
   <property name="toolTip">
    <string>Add Cable</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionPVBar">
   <property name="icon">
    <iconset resource="PnGraphics/pnrc.qrc">
     <normaloff>:/img/ico-pv.png</normaloff>:/img/ico-pv.png</iconset>
   </property>
   <property name="text">
    <string>PVBar</string>
   </property>
   <property name="toolTip">
    <string>Add PVBar</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PnView</class>
   <extends>QWidget</extends>
   <header location="global">PnGraphics/pnview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PnGraphics/pnrc.qrc"/>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
